treeherder/seta/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
```

Comment:
I *think* I know where this is going.

* Drop urls.py
* Drop views.py (since I have nothing useful there)
* Build seta APIs ala performance_data.py (e.g. webapp/api/seta.py)

I will look at the rest_framework tutorials/documentation and focus on a basic hello world api to see if I'm going in the right direction.